odoo/addons/base/models/res_country.py

```diff
@@ -69,7 +69,7 @@ class Country(models.Model):
             ('after', 'After Address'),
         ], string="Customer Name Position", default="before",
         help="Determines where the customer/company name should be placed, i.e. after or before the address.")
-    vat_label = fields.Char(string='Vat Label', translate=True, help="Use this field if you want to change vat label.")
+    vat_label = fields.Char(string='Vat Label', translate=True, prefetch=True, help="Use this field if you want to change vat label.")
```

Comment:
This one is a bit surprising at first sight.  Is this field necessary for regular business code?

Reply:
If I remember, it is for the method `value_to_html` and for the usage in lot of report.